tests/test_acs_and_reviewers.py

```diff
@@ -0,0 +1,123 @@
+import os
+import re
+import csv
+import pytest
+import random
+import datetime
+import re
+import openreview
+from openreview.api import Note
+from selenium.webdriver.common.by import By
+from openreview.api import OpenReviewClient
+from openreview.workflows import templates
+from openreview.workflows import workflows
+
+class TestSimpleDualAnonymous():
+
+    def test_setup(self, openreview_client, helpers):
+        super_id = 'openreview.net'
+        support_group_id = super_id + '/Support'
+
+        helpers.create_user('programchair@efgh.cc', 'ProgramChair', 'EFGH')
+        helpers.create_user('reviewer_one@efgh.cc', 'ReviewerOne', 'EFGH')
+        helpers.create_user('reviewer_two@efgh.cc', 'ReviewerTwo', 'EFGH')
+        helpers.create_user('reviewer_three@efgh.cc', 'ReviewerThree', 'EFGH')
+        helpers.create_user('areachair_one@efgh.cc', 'ACOne', 'EFGH')
+        helpers.create_user('areachair_one_two@efgh.cc', 'ACTwo', 'EFGH')
+        helpers.create_user('areachair_one_three@efgh.cc', 'ACThree', 'EFGH')
+        pc_client=openreview.api.OpenReviewClient(username='programchair@efgh.cc', password=helpers.strong_password)
+
+        workflows_setup = workflows.Workflows(openreview_client, support_group_id, super_id)
+        workflows_setup.setup()
+
+        templates_invitations = templates.Templates(openreview_client, support_group_id, super_id)
+        templates_invitations.setup()
+
+        assert openreview_client.get_invitation('openreview.net/-/Edit')
+        assert openreview_client.get_group('openreview.net/Support/Venue_Request')
+        assert openreview_client.get_group('openreview.net/Support/Venue_Request/ACs_and_Reviewers')
+        assert openreview_client.get_invitation('openreview.net/Support/Venue_Request/-/ACs_and_Reviewers')
+        assert openreview_client.get_invitation('openreview.net/Support/Venue_Request/ACs_and_Reviewers/-/Deployment')
+        assert openreview_client.get_invitation('openreview.net/Support/Venue_Request/ACs_and_Reviewers/-/Comment')
+
+        now = datetime.datetime.now()
+        due_date = now + datetime.timedelta(days=2)
+
+        request = pc_client.post_note_edit(invitation='openreview.net/Support/Venue_Request/-/ACs_and_Reviewers',
+            signatures=['~ProgramChair_EFGH1'],
+            note=openreview.api.Note(
+                content={
+                    'official_venue_name': { 'value': 'The EFGH Conference' },
```

Comment:
could we add two fields to specify the reviewers_name and area_chairs_name?

Reply:
Done.